### PR TITLE
Update layout of new editor format bar on iPads

### PR DESCRIPTION
**Pull request by bummytime**
Fixes part of https://github.com/wordpress-mobile/WordPress-iOS-Editor/issues/65

iPad format bar should now look like this:

scaled screen shot 2015-01-08 at 1 36 10 pm

iPhone format bars will look the same.

@SergioEstevao or @aerych would you mind taking a peek at this for me?

### CODE REVIEW

**Comment by aerych** [comment]
> iPhone format bars will look the same.

iPhone format bars will be unchanged?

File: Classes/WPEditorToolbarView.m [hunk 1]
```diff
@@ -7,6 +7,7 @@
 static int kDefaultToolbarLeftPadding = 10;
 
 static int kNegativeToolbarItemPadding = 12;
+static int kiPadNegativeToolbarItemPadding = 10;
```
Comment (aerych) [inline Classes/WPEditorToolbarView.m]: Xcode is flagging this as an unused variable

File: Assets/ZSSRichTextEditor.js [hunk 2]
```diff
@@ -48,6 +51,16 @@ ZSSEditor.defaultParagraphSeparator = 'p';
...
 ZSSEditor.init = function() {
     
+    // Change a few CSS values if the device is an iPad
+    ZSSEditor.isiPad = (navigator.userAgent.match(/iPad/i) != null);
+    if (ZSSEditor.isiPad) {
+        $(document.body).css('padding-left', '80px');
+        $(document.body).css('padding-right', '80px');
+        $('#zss_field_title').css('font-size', '1.3em');
+        $('#zss_field_content').css('font-size', '1.125em');
+        $('#zss_field_content').css('line-height', '28px');
```
Comment (aerych) [inline Assets/ZSSRichTextEditor.js]: Would it be make sense to move these styles into the css file, and then update `document.body.className` with the relevant class for a true IF check?

**Comment by aerych** [comment]
I'm wondering if it would be good to frame this not in terms of iPhone vs iPad but in terms of available screen-size. This might anticipate the day we can use the size classes introduced in iOS 8, but in the short term it could benefit owners of the iPhone 6+ and its monster screen.

**Comment by aerych** [comment]
Btw, I dig the spacing and grouping of the toolbar buttons on the iPad. They look nice and I'm kind of wondering if the landscape iPhone toolbar could take advantage of the spacing too. :D

File: Assets/ZSSRichTextEditor.js (hunk 2 above)
Comment (bummytime) [inline Assets/ZSSRichTextEditor.js]: Make sense to me...change forth coming

**Comment by bummytime** [comment]
> iPhone format bars will be unchanged?

That is correct

> I'm wondering if it would be good to frame this not in terms of iPhone vs iPad but in terms of available screen-size.

I thought the same thing as well, but ultimately decided on this approach because things will be changing with Calypso soonish.

**Comment by bummytime** [comment]
>Btw, I dig the spacing and grouping of the toolbar buttons on the iPad. They look nice and I'm kind of wondering if the landscape iPhone toolbar could take advantage of the spacing too. :D

Let me explore that and get back to ya. :-)

File: Classes/WPEditorToolbarView.m (hunk 1 above)
Comment (bummytime) [inline Classes/WPEditorToolbarView.m]: I will fix

**Comment by thianhlu** [comment]
Not enough space on iPhone to do this right? Yeah logical grouping is nice.

**Comment by bummytime** [comment]
@thianhlu yeah, ultimately there isn't any room to do this on smaller devices.

**Comment by bummytime** [comment]
@aerych Changes made. Would you mind taking one moar quick peek?

**Comment by aerych** [comment]
Took it for a quick test drive.  Looks good :)  :shipit:

**Pull request merged**